atmosphere/plugins/quota/default_quota.py

```diff
@@ -0,0 +1,4 @@
+class DefaultQuotaPlugin(object):
+    def get_default_quota(self, user, provider):
+        raise NotImplementedError("Validation plugins must implement a get_default_quota function that "
+                                  "takes two arguments: 'user' and 'provider")
```

Comment:
As discussed, default is _no plugin_. It will then just use the default quotas as specified in the settings.